### PR TITLE
Add better handling of open and closed contours. #330

### PR DESCRIPTION
**Pull request by nbassler**

### CODE REVIEW

File: pytrip/vdx.py [hunk 1]
```diff
@@ -1336,11 +1344,24 @@ def read_vdx(self, content, i):
                     points = int(line.split()[1])
                     set_point = True
             i += 1
+
+        # check if the contour is closed
+        if len(self.contour > 1):  # check if this is an actual contour, and not a POI
```
Comment (grzanka) [inline pytrip/vdx.py]: wrong brackets ?

File: pytrip/vdx.py [hunk 2]
```diff
@@ -1359,6 +1380,15 @@ def read_vdx_old(self, slice_number, xy_line):
                                  _pixel_size * float(y) / 16.0,
                                  float(slice_number)])
 
+        # check if the contour is closed
+        if len(self.contour > 1):  # check if this is an actual contour, and not a POI
```
Comment (grzanka) [inline pytrip/vdx.py]: wrongly placed brackets ?

File: pytrip/vdx.py (hunk 1 above)
Comment (nbassler) [inline pytrip/vdx.py]: its WIP, i will add you as reviewer when needed... ;)

**Comment by nbassler** [comment]
This was tested with branch https://github.com/pytrip/pytripgui/tree/feature/72-closed

**Comment by nbassler** [comment]
@grzanka now ready for review :)

**Pull request merged**